### PR TITLE
Simplify README and add cloud deployment

### PR DESCRIPTION
**Pull request by chris-crone**
Simplify the README and add links to the cloud integrations.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,62 +1,86 @@
...
 ![Docker Compose](logo.png?raw=true "Docker Compose Logo")
 
-Compose is a tool for defining and running multi-container Docker applications.
-With Compose, you use a Compose file to configure your application's services.
-Then, using a single command, you create and start all the services
-from your configuration. To learn more about all the features of Compose
-see [the list of features](https://github.com/docker/docker.github.io/blob/master/compose/index.md#features).
+Docker Compose is a tool for running multi-container applications on Docker
+defined using the [Compose file format](https://compose-spec.io).
+A Compose file is used to define how the one or more containers that make up
```
Comment (gtardif) [inline README.md:9]:
```suggestion
A Compose file is used to define how the container(s) that make up
```

File: README.md [hunk 2]
```diff
@@ -1,62 +1,86 @@
...
+your application are configured.
+Once you have a Compose file, you can create and start your application with a
+single command: `docker-compose up`.
 
-Compose is great for development, testing, and staging environments, as well as
-CI workflows. You can learn more about each case in
-[Common Use Cases](https://github.com/docker/docker.github.io/blob/master/compose/index.md#common-use-cases).
+Compose files can be used to deploy applications to
```
Comment (gtardif) [inline README.md:14]:
```suggestion
Compose files can be used to deploy applications locally, or to
```

**Review by gtardif — APPROVED** [review]
a couple of nit suggestions, otherwise LGTM

File: README.md (hunk 1 above)
Comment (chris-crone) [reply README.md:9]: I wanted to call out one container here as we see Compose as a way to script `docker run` as well.

File: README.md (hunk 2 above)
Comment (chris-crone) [reply README.md:14]: Good catch, will fix.

**Pull request merged**